pkg/sysinfo/sysinfo_linux.go

```diff
@@ -32,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		sysInfo.AppArmor = true
 	}
 
+	// Check if Seccomp is supported.
+	if _, _, err := syscall.RawSyscall(syscall.SYS_PRCTL, syscall.PR_GET_SECCOMP, 0, 0); err != syscall.EINVAL {
```

Comment:
Ya idk which one is the lesser of two evils, I'm down for whatever everyone thinks is best


On Jan 11, 2016, 21:07 -0800, Tianon Gravi<notifications@github.com>, wrote:
>  
> Inpkg/sysinfo/sysinfo_linux.go(https://github.com/docker/docker/pull/19245#discussion_r49416374):
>  
> >@@ -32,6 +33,11 @@ func New(quiet bool) *SysInfo {>sysInfo.AppArmor = true>}>>+ // Check if Seccomp is supported.>+ if _, _, err := syscall.RawSyscall(syscall.SYS_PRCTL, syscall.PR_GET_SECCOMP, 0, 0); err != syscall.EINVAL {
> Yeah, makes sense, but those docs are definitely scary lol - Tianon
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/docker/docker/pull/19245/files#r49416374).
>  
>  
>